api/v1beta1/hyperconverged_types.go

```diff
@@ -32,6 +32,7 @@ type HyperConvergedTuningPolicy string
 const (
 	HyperConvergedAnnotationTuningPolicy HyperConvergedTuningPolicy = "annotation"
 	HyperConvergedHighBurstProfile       HyperConvergedTuningPolicy = "highBurst"
+	HyperConvergedKubeletProfile         HyperConvergedTuningPolicy = "kubelet"
```

Comment:
What about `follow-kubelet`

Reply:
Sounds good to me! We have also considered naming it as something like `mcoKubelet`, with the idea of reflecting the dependency of the machine configuration operator (mco). Another possibility is a merge between both ideas like: `followMCOKubelet`. WDYT? In any case, `followKubelet` (IMHO without the `-` for consistency purposes) sounds better than just `kubelet` :)